### PR TITLE
Rename stocks to equity

### PR DESCRIPTION
**Pull request by piiq**

### CODE REVIEW

File: openbb_platform/extensions/equity/openbb_equity/estimates/estimates_router.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    provider_choices: ProviderChoices,
+    standard_params: StandardParams,
+    extra_params: ExtraParams,
+) -> OBBject[BaseModel]:
+    """Price Target. Price target data."""
+    return OBBject(results=Query(**locals()).execute())
+
+
+@router.command(model="AnalystEstimates")
+def analyst(
```
Comment (jmaslek) [inline openbb_platform/extensions/equity/openbb_equity/estimates/estimates_router.py:30]: should we be more verbose: analyst_estimates?

File: openbb_platform/extensions/equity/openbb_equity/estimates/estimates_router.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    provider_choices: ProviderChoices,
+    standard_params: StandardParams,
+    extra_params: ExtraParams,
+) -> OBBject[BaseModel]:
+    """Analyst Estimates. Analyst stock recommendations."""
+    return OBBject(results=Query(**locals()).execute())
+
+
+@router.command(model="PriceTargetConsensus")
+def consensus(
```
Comment (jmaslek) [inline openbb_platform/extensions/equity/openbb_equity/estimates/estimates_router.py:41]: Same here.  price_target_consensus ?

File: openbb_platform/extensions/equity/openbb_equity/fundamental/fundamental_router.py [hunk 3]
```diff
@@ -93,18 +82,18 @@ def divs(
 
 
 @router.command(model="EarningsCalendar")
-def earning(
+def earnings(
```
Comment (jmaslek) [inline openbb_platform/extensions/equity/openbb_equity/fundamental/fundamental_router.py:85]: Whats different here and the earnings calendar above?

File: openbb_platform/extensions/equity/openbb_equity/fundamental/fundamental_router.py [hunk 4]
```diff
@@ -93,18 +82,18 @@ def divs(
...
     extra_params: ExtraParams,
 ) -> OBBject[BaseModel]:
-    """Earnings Calendar. Earnings calendar for a given company."""
+    """Historical Earnings for a given company."""
     return OBBject(results=Query(**locals()).execute())
 
 
 @router.command(model="HistoricalEmployees")
-def emp(
+def employees(
```
Comment (jmaslek) [inline openbb_platform/extensions/equity/openbb_equity/fundamental/fundamental_router.py:96]: maybe something like employee_count ?

File: openbb_platform/extensions/equity/openbb_equity/fundamental/fundamental_router.py [hunk 5]
```diff
@@ -279,19 +213,8 @@ def filings(
...
-    standard_params: StandardParams,
-    extra_params: ExtraParams,
-) -> OBBject[BaseModel]:
-    """Share Statistics. Share statistics for a given company."""
-    return OBBject(results=Query(**locals()).execute())
-
-
 @router.command(model="HistoricalStockSplits")
-def split(
+def splits(
```
Comment (jmaslek) [inline openbb_platform/extensions/equity/openbb_equity/fundamental/fundamental_router.py:217]: same -- what is different from above?

File: openbb_platform/extensions/equity/openbb_equity/shorts/shorts_router.py [hunk 6]
```diff
@@ -10,13 +10,13 @@
...
 
-router = Router(prefix="/dps")
+router = Router(prefix="/shorts")
 
 # pylint: disable=unused-argument
 
 
 @router.command(model="StockFTD")
-def ftd(
+def fail_to_deliver(
```
Comment (jmaslek) [inline openbb_platform/extensions/equity/openbb_equity/shorts/shorts_router.py:19]: fails_to_deliver

File: openbb_platform/extensions/equity/openbb_equity/estimates/estimates_router.py (hunk 2 above)
Comment (deeleeramone) [reply openbb_platform/extensions/equity/openbb_equity/estimates/estimates_router.py:41]: WDYT about, "analyst_consensus"?

File: openbb_platform/extensions/equity/openbb_equity/fundamental/fundamental_router.py (hunk 3 above)
Comment (deeleeramone) [reply openbb_platform/extensions/equity/openbb_equity/fundamental/fundamental_router.py:85]: This is not actually an earnings calendar.  This function actually returns,`historical_eps`

File: openbb_platform/extensions/equity/openbb_equity/fundamental/fundamental_router.py (hunk 5 above)
Comment (deeleeramone) [reply openbb_platform/extensions/equity/openbb_equity/fundamental/fundamental_router.py:217]: There are `historical_splits`, by ticker, and then there should be a different calendar endpoint that does not go in the FA router.  The function `compsplit()` should be taken out of FA.

File: openbb_platform/extensions/equity/openbb_equity/calendar/calendar_router.py [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+
+
+@router.command(model="EarningsCalendar")
+def earnings(
+    cc: CommandContext,
+    provider_choices: ProviderChoices,
+    standard_params: StandardParams,
+    extra_params: ExtraParams,
+) -> OBBject[BaseModel]:
+    """Earnings Calendar. Earnings calendar for a given company."""
```
Comment (deeleeramone) [inline openbb_platform/extensions/equity/openbb_equity/calendar/calendar_router.py:58]: """Upcoming and Historical earnings calendar."""

File: openbb_platform/extensions/equity/openbb_equity/discovery/discovery_router.py [hunk 8]
```diff
@@ -11,7 +11,7 @@
 from openbb_core.app.router import Router
 from pydantic import BaseModel
 
-router = Router(prefix="/disc")
+router = Router(prefix="/discovery")
```
Comment (deeleeramone) [inline openbb_platform/extensions/equity/openbb_equity/discovery/discovery_router.py:14]: `equity_discovery_undervalued_growth_equities` could trim`equities` from the end.

File: openbb_platform/extensions/equity/openbb_equity/estimates/estimates_router.py (hunk 1 above)
Comment (deeleeramone) [reply openbb_platform/extensions/equity/openbb_equity/estimates/estimates_router.py:30]: This function is probably better described by, `historical_earnings_consensus`

Screenshot 2023-11-08 at 2 03 25 PM

Comment (piiq) [reply openbb_platform/extensions/equity/openbb_equity/estimates/estimates_router.py:30]: how about `obb.equity.estimates.historical` here? then the other one can stay `obb.equity.estimates.consensus`

File: openbb_platform/extensions/equity/openbb_equity/fundamental/fundamental_router.py (hunk 3 above)
Comment (piiq) [reply openbb_platform/extensions/equity/openbb_equity/fundamental/fundamental_router.py:85]: still need a decision on this one. There is a bug on the models side where the model does not return what it should and yes there is a separate calendar `obb.equity.calendar.earnings` command

File: openbb_platform/extensions/equity/openbb_equity/discovery/discovery_router.py (hunk 8 above)
Comment (piiq) [reply openbb_platform/extensions/equity/openbb_equity/discovery/discovery_router.py:14]: image

**Comment by piiq** [comment]
So i've fixed a few bugs and updated the tests. I've uncovered failing integration tests on the web api side in the discovery menu. Also the comments that have not been resolved are not resolved. @IgorWounds please give some feedback (see discussions above) and it will be then ready to merge

**Comment by piiq** [comment]
not exactly. the tests that rely on the tests fail. fixing

File: openbb_platform/extensions/equity/openbb_equity/fundamental/fundamental_router.py (hunk 3 above)
Comment (piiq) [reply openbb_platform/extensions/equity/openbb_equity/fundamental/fundamental_router.py:85]: IIUC this question will be addressed in this PR
https://github.com/OpenBB-finance/OpenBBTerminal/pull/5690

That PR will also need to address the renaming of the historical earnings model

File: openbb_platform/extensions/equity/openbb_equity/estimates/estimates_router.py (hunk 1 above)
Comment (piiq) [reply openbb_platform/extensions/equity/openbb_equity/estimates/estimates_router.py]: i've put `obb.equity.estimates.historical` and `obb.equity.estimates.consensus`. We can re-think this in another PR

File: openbb_platform/extensions/equity/openbb_equity/estimates/estimates_router.py (hunk 2 above)
Comment (piiq) [reply openbb_platform/extensions/equity/openbb_equity/estimates/estimates_router.py:41]: see comment above

**Comment by piiq** [comment]
it's good from my side. i've identified a few flaky api tests, but they should not blocking this merge

**Comment by deeleeramone** [comment]
> it's good from my side. i've identified a few flaky api tests, but they should not blocking this merge

So, when an item, like a symbol directory, the cassettes will not record a cached request event.  If functions share a cached resource, it will only capture the cassette for the first instance.  The workaround is to delete the cache file and then only run the single test which needs to be recorded.

I've captured the missing cassettes and commented out the `skip`.

**Pull request merged**